Formula/x/xmq.rb

```diff
@@ -0,0 +1,40 @@
+class Xmq < Formula
+  desc "Tool and language to work with xml/html/json"
+  homepage "https://libxmq.org"
+  url "https://github.com/libxmq/xmq/archive/refs/tags/3.1.3.tar.gz"
+  sha256 "f58dc5a1d3c523e19a9db24a7d14dd96e5307425950b0fefeae94d3c2ccc7339"
+  license "MIT"
+
+  head do
+    url "https://github.com/libxmq/xmq.git", branch: "master"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+  end
+
+  depends_on "pkgconf" => :build
+
+  on_linux do
+    depends_on "libxml2"
```

Comment:
can be replaced with 
```suggestion
  uses_from_macos "libxml2"
```